packages/account/src/Sections/Assessment/FinancialAssessment/financial-assessment.tsx

```diff
@@ -116,7 +135,7 @@ const ConfirmationPage = ({ toggleModal, onSubmit }) => (
     </div>
 );
 
-const SubmittedPage = ({ platform, routeBackInApp }) => {
+const SubmittedPage = ({ platform, routeBackInApp }: TSubmittedPage) => {
```

Comment:
@likhith-deriv You can use `useStore` to get `routeBackInApp` instead of passing prop.